File: PooCC01CNetz/Semana02/Python/Ejercicios1/Empleado.py
class Empleado:
    # Declaro variables en la clase
    def __init__(self):
        self.nombre = ""
        self.edad = 0
        self.salario = 0.0

    # Declaro un metodo
    def calculaISR(self):
        isr = self.salario * 30 * 0.15
        return isr

File: PooCC01CNetz/Semana02/Java/Ejercicios1/Empleado.java
public class Empleado {
    // Declaro variables en la clase
    public String nombre;
    public int edad;
    public double salario;

    // Declaro un metodo
    public double calculaISR(){
        double isr = salario * 30 * 0.15;
        return isr;
    }
}
